frontend/src/common/entities.ts

```diff
@@ -0,0 +1,109 @@
+export interface Project {
+  // assays: string[];
+  // biosample_categories: string[];
+  // cell_count: number;
+  // contributors: Contributor[];
+  // cxg_enabled: boolean;
+  // description: string;
+  // diseases: string[];
+  // id: string;
+  // label: string;
+  // organs: string[];
+  // paired_end: string[];
+  // publication_title: string;
+  // species: string[];
+  // name: string;
+  // owner: {
+  //   id: string;
+  //   email: string;
+  // };
+  // status: string;
+  // processing_state: string;
+  // s3_bucket_key: string;
+  // validation_state: string;
+  // links: { url: string; type: string }[];
+  // attestation: {
+  //   needed: boolean;
+  //   tc_uri: string;
+  // };
+  datasets: Dataset[];
+  links: { name: string; url: string }[];
```

Comment:
`links` does not contain `name` yet. Will this work when integrated?

Reply:
Thanks for raising this! I just pushed a commit to temporarily comment out `<MoreInformation />` component, which is the only one that uses `link.name`. So the type definition can stay!